### ISSUE TITLE
Add option to setup a default calendar

### ISSUE THREAD

**Issue by Mannshoch** [SimpleMobileTools/Simple-Calendar#464]
I use around 10 CalDav calendar. But only two of them do I need for new events.

**Comment by tibbi** [comment]
Well the calendar you used the last time is set as the default in the next event, no?

**Comment by Mannshoch** [comment]
Hmm... to much experimenting with my calendars. i didnt realized the good idea for using the last used one. 
I like this tool more and more 👍👍

**Comment by Mannshoch** [comment]
Sorry for that but I stumbled into a Problem about my habit. I use this Calendar now often but use regularly one specific calendar. I have no problem with changing the Calendar if i have a special entry. But I forget to change it if I add again like normal. So it's happened twice that I filled the wrong calendar with my personal dates. 
I reopen this Ticket because I need a possibility to set up a standard calendar. Maybe if you like It, a way to limit to auto remember the last calendar only on some specific calendar would also be a solution. The Idea here is to be sure that some calendar have to be Opt-in calendars. Such Opt-in calendars in my case are a calendar that add an entry into a website or entries that are shared with third persons. Calendars that not even accidental should get filled with Data.
